src/cloud-api-adaptor/podvm-mkosi/mkosi.presets/system/mkosi.repart/30-media.conf

```diff
@@ -0,0 +1,5 @@
+[Partition]
```

Comment:
Why do we create a partition? we just need a folder on the existing root partition, no? we can't create that adhoc at runtime, because it's a read-only fs, but a folder in ./podvm/files should work, I guess

```
[root@fedora ~]# mount /dev/sr0 /media
mount: /media: WARNING: source write-protected, mounted read-only.
[root@fedora ~]# ls -l /media/
total 5
-rw-r--r-- 1 1000 1000    0 Nov 22 14:10 meta-data
-rw-r--r-- 1 1000 1000 4584 Jan  9 09:57 user-data
```

Reply:
Ah yes :-) ..